Gruntfile.js

```diff
@@ -58,7 +58,7 @@ module.exports = function (grunt) {
         src: ['./client.coffee'],
         dest: 'client/client.max.js',
         options: {
-          transform: [['coffeeify', {transpile: {presets: ['@babel/preset-env']}}]],
+          transform: [['coffeeify', {transpile: {presets: ['@babel/preset-env'], plugins: ['@babel/plugin-transform-runtime']}}]],
```

Comment:
We were already using Babel? Now using a runtime? What should programmers know about this?